kotlin/openID4VP/src/main/java/io/mosip/openID4VP/authorizationRequest/ClientMetadata.kt

```diff
@@ -16,41 +18,63 @@ private val className = ClientMetadata::class.simpleName!!
 object ClientMetadataSerializer : KSerializer<ClientMetadata> {
 	override val descriptor: SerialDescriptor = buildClassSerialDescriptor("ClientMetadata") {
 		element<String>("name")
```

Comment:
it should be client_name https://www.rfc-editor.org/rfc/rfc7591.txt
consider name as optional and fallback to client-id

Reply:
done